Tools/CMake/AMReX_Config.H.in

```diff
@@ -58,7 +58,7 @@
 #ifdef __cplusplus@COMP_DECLS@
 @OMP_DECLS@
 #endif
-#if defined(AMREX_USE_OMP) && !defined(_OPENMP)
+#if defined(AMREX_USE_OMP) && !defined(_OPENMP) && !defined(AMREX_USE_HIP)
```

Comment:
Could we add a comment on why?  We might forget about it in the future.

Reply:
Added comment